### PR TITLE
Fix import eDistantObject for cocoapods

### PR DESCRIPTION
**Pull request by konradzdunczyk**
Fix for #1510

### CODE REVIEW

File: CommonLib/DistantObject/GREYHostApplicationDistantObject.h [hunk 1]
```diff
@@ -16,11 +16,11 @@
 
 #import <Foundation/Foundation.h>
 
-// Use modular imports for CocoaPods release.
-// #import <eDistantObject/EDOClientService.h>
-// End CocoaPods modular imports
-
+#if COCOAPODS
```
Comment (tirodkar) [inline CommonLib/DistantObject/GREYHostApplicationDistantObject.h:19]: Is this a define CocoaPods provides us?

Comment (konradzdunczyk) [reply CommonLib/DistantObject/GREYHostApplicationDistantObject.h:19]: Yes

image

**Comment by shaneong** [comment]
Please merge this PR as it is blocking our use case where we have a separate Pod Framework in our Podfile hence EarlGrey is failing to build without this fix at the moment.

**Comment by shaneong** [comment]
How long does it take for changes to get merged in to CocoaPods? Am still not observing this change in installed EarlGreyTest Pod

**Comment by tirodkar** [comment]
@shaneong we merged this internally so closing this. We'll have to do another CocoaPods release which I can check with the team on the duration of.

**Comment by shaneong** [comment]
Can you prioritize this fix as being able to use EarlGrey in conjunction with use_frameworks! should be a common ask in CocoaPods, or at least provide a workaround solution in the meantime?

**Pull request closed without merging**